pkg/collector/corechecks/net/networkv2/network.go

```diff
@@ -162,11 +175,7 @@ func (c *NetworkCheck) Run() error {
 	if c.config.instance.CollectConnectionState {
 		netProcfsBasePath := c.net.GetNetProcBasePath()
 		for _, protocol := range []string{"udp4", "udp6", "tcp4", "tcp6"} {
-			connectionsStats, err := c.net.Connections(protocol)
```

Comment:
before we were incorrectly using gopsutil instead of socket stat/ netstat to get connection states which missed some